client/state/media/actions.js

```diff
@@ -80,13 +84,15 @@ export function requestingMedia( siteId, query ) {
  *
  * @param  {number} siteId Site ID
  * @param  {object} query  Query object
+ * @param  {object} error  Error object
  * @returns {object}        Action object
  */
-export function failMediaRequest( siteId, query ) {
+export function failMediaRequest( siteId, query, error = null ) {
```

Comment:
why null vs leaving it `undefined` if it's not set?

Reply:
Because once we start using those errors in #39375 dealing with an undefined was throwing an error.